pkg/kubelet/allocation/allocation_manager_test.go

```diff
@@ -204,72 +214,83 @@ func TestIsPodResizeInProgress(t *testing.T) {
 	}
 
 	tests := []struct {
-		name            string
-		containers      []testContainer
-		expectHasResize bool
+		name                            string
+		containers                      []testContainer
+		oldPodResizeInProgressCondition bool
+		expectHasResize                 bool
+		expectPodResizeCompletedMsg     string
 	}{{
 		name: "simple running container",
 		containers: []testContainer{{
 			allocated: testResources{100, 100, 100, 100},
 			actuated:  &testResources{100, 100, 100, 100},
 			isRunning: true,
 		}},
-		expectHasResize: false,
+		oldPodResizeInProgressCondition: true,
+		expectHasResize:                 false,
+		expectPodResizeCompletedMsg:     "Normal ResizeCompleted Pod resize completed, {\"containers\":[{\"Name\":\"c0\",\"Resources\":{\"limits\":{\"cpu\":\"100m\",\"memory\":\"100\"},\"requests\":{\"cpu\":\"100m\",\"memory\":\"100\"}}}]}",
```

Comment:
nit: this is more readable if you use a raw string

```suggestion
		expectPodResizeCompletedMsg:     `Normal ResizeCompleted Pod resize completed, {"containers":[{"Name":"c0","Resources":{"limits":{"cpu":"100m","memory":"100"},"requests":{"cpu":"100m","memory":"100"}}}]}`,
```

Reply:
same below